src/lib/emailRepository.ts

```diff
@@ -298,13 +298,14 @@ class EmailRepository {
 		}
 	}
 
-	public async updateEmailIsSetup(userId: string, val: boolean) {
+	public updateEmailIsSetup(userId: string, val: boolean) : RunResult {
```

Comment:
This shouldn't return run result but the result of the run mapped to a type within our domain

Reply:
And async

Reply:
This does not actually return anything so no need to return antyhing